attempt to fix undefined references to dcsctp by removing dcsctp_transport.*
https://github.com/desktop-app/tg_owt/pull/94

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -827,8 +827,6 @@ PRIVATE
     media/engine/webrtc_media_engine_defaults.cc
     media/engine/webrtc_video_engine.cc
     media/engine/webrtc_voice_engine.cc
-    media/sctp/dcsctp_transport.cc
-    media/sctp/dcsctp_transport.h
     media/sctp/sctp_transport_factory.cc
     media/sctp/sctp_transport_factory.h
     media/sctp/usrsctp_transport.cc
